File: Guia1_2022_Prueba/app/src/main/java/com/example/guia1_2022_prueba/SplashScreenActivity.java
package com.example.guia1_2022_prueba;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;

public class SplashScreenActivity extends AppCompatActivity {
    public static int SPLAH_TIME_OUT = 100;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        /*setContentView(R.layout.activity_splash_screen);
        startActivity(new Intent(SplashScreenActivity.this, MainActivity.class));*/

        new Handler().postDelayed(new Runnable()
        {
            @Override
            public void run() {
                startActivity(new Intent(SplashScreenActivity.this, MainActivity.class));
                finish();
            }
        }, SPLAH_TIME_OUT);
    }
}

File: CalculadoraKotlin/app/src/main/java/com/example/calculadorakotlin/SplashScreenActivity.kt
package com.example.calculadorakotlin

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class SplashScreenActivity : AppCompatActivity() {
    private var SPLAHTIMEOUT: Long = 100
    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        Handler().postDelayed(
            {
                fun run() {
                    startActivity(Intent(this@SplashScreenActivity, MainActivity::class.java))
                    finish()
                }
            },   SPLAHTIMEOUT     )

    }
}